DNN Platform/Library/Entities/Portals/PortalInfo.cs

```diff
@@ -827,7 +800,6 @@ public void Fill(IDataReader dr)
             PaymentProcessor = Null.SetNullString(dr["PaymentProcessor"]);
             ProcessorUserId = Null.SetNullString(dr["ProcessorUserId"]);
             ProcessorPassword = Null.SetNullString(dr["ProcessorPassword"]);
-            SiteLogHistory = Null.SetNullInteger(dr["SiteLogHistory"]);
```

Comment:
Ideally, we should review the SProcs that returns SiteLogHistory field to remove that field